README.md

```diff
@@ -14,7 +14,8 @@ Release | Notes
 
 Project | Aras
 --------|------
-[v1](https://github.com/ArasLabs/start-specific-workflow/releases/tag/v1) | 9.2.0 SP3
+
+[v1](https://github.com/ArasLabs/start-specific-workflow/releases/tag/v1) | Aras 11 SP12
```

Comment:
We'll create a new release (v2.0.0) for this update since there is a change to the code. Both release tables need a new row for the new release, like this:
```suggestion
[v2.0.0](https://github.com/ArasLabs/start-specific-workflow/releases/tag/v2.0.0) | Aras 11 SP12
[v1](https://github.com/ArasLabs/start-specific-workflow/releases/tag/v1) | 9.2.0 SP3
```

Reply:
@sampoearas will make these changes after the merge.